### ISSUE TITLE
New a function to create ad-hoc osmconf.ini files

### ISSUE THREAD

**Issue by agila5** [ITSLeeds/osmextract#140]
At the moment this is hard-coded in 

https://github.com/ITSLeeds/osmextract/blob/e81b985f3f4f99bbb1473cb4b8c95b7d00578aa6/R/vectortranslate.R#L260-L281

but I think it could be nice to export a new function that has the same goals, thoughts?

**Comment by Robinlovelace** [comment]
Yes, I think it would be useful to have a helper function that opens the default .ini file and maybe also prints a message with a link to info on how to edit it.
